documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -2589,13 +2564,6 @@ Hashing strategy version 2 should be used to ensure fidelity if the value is bei
 |bytes
 |Specifies how binary (`blob`) columns should be represented in change events, including: `bytes` represents binary data as byte array (default), `base64` represents binary data as base64-encoded String, `hex` represents binary data as hex-encoded (base16) String
 
-|[[oracle-property-schema-name-adjustment-mode]]<<oracle-property-schema-name-adjustment-mode,`+schema.name.adjustment.mode+`>>
-|avro
-|Specifies how schema names should be adjusted for compatibility with the message converter used by the connector. Possible settings:  +
-
-* `avro` replaces the characters that cannot be used in the Avro type name with underscore. +
-* `none` does not apply any adjustment. +
-
```

Comment:
This should not have been removed.

Reply:
I didn't remove this intentionally. Should I re-add the content and push the change?